day-2/advice/src/App.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Nav from './components/Nav';
+import ReactDOM from 'react-dom';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import RaisedButton from 'material-ui/RaisedButton';
+import StickyFooter from 'material-ui/BottomNavigation';
+
+
+var quotes = ['Kick ass today', 
+              'Act like you know what you\'re doing',
+              'Watch out for the fucking scope creep',
+              'Make sure there\'s money in the bank',
+              'Get that git flow', 
+              ]
+var _this;
+
+class App extends Component {
+    constructor() {
+      super();
+      _this = this;
+      this.state = {
+        index:0 
+      };
+    }
+  render() {
+    return (
+      <div className="App">
+        <div className="App-header">
+          <h2>Great fucking advice</h2>
+        </div>
+
+        <div className="Nav"> 
+          <MuiThemeProvider>
```

Comment:
`MuiThemeProvider` should be a top level element in the dom tree. see [usage instructions](https://github.com/callemall/material-ui#usage) for more details. It should not be repeatedly used as a parent